var/spack/repos/builtin/packages/py-bglibpy/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyBglibpy(PythonPackage):
+    """Pythonic Blue Brain simulator access"""
+    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/sim/BGLibPy"
+    git      = "ssh://bbpcode.epfl.ch/sim/BGLibPy"
+
+    version('develop', branch='master')
+    version('4.0.17', commit='e90513f52a7d9ca3c16877ae7a2bcb8df31c8545', preferred=True)
+
+    depends_on('py-setuptools', type=('build', 'run'))
+
+    depends_on('neuron+python', type='run')
+    depends_on('py-h5py~mpi@2.3:', type='run')
```

Comment:
Why do we need to disable mpi here ?

Reply:
I kept copy-pasting this line across the recipes, except the cases where I was sure MPI is needed.
If we have `py-h5py` variant with MPI pre-deployed (not sure it was the case from the start), I guess we can remove `~mpi` here as well.

Reply:
I "tried" to change this default in upstream, see https://github.com/spack/spack/pull/9321.

Depends on bglibpy use case : if it's going to be used in serial mode, doesn't necessary to build build mpi and all dependencies (e.g. non-bb5 system where we don't have preinstalled packages)

Reply:
@wvangeit, are you OK if we keep `~mpi` now and relax it later if needed? (we'll need to do the same for `bluepy` at that moment).